compiler/damlc/tests/src/DamlcVisual.hs

```diff
@@ -1,39 +0,0 @@
--- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
--- SPDX-License-Identifier: Apache-2.0
-module VisualTest
-   ( main
-   ) where
-
-import Test.Tasty
-import System.IO.Extra
-import DA.Cli.Visual
-import DA.Daml.LF.Reader
-import Test.Tasty.Golden
-import DA.Bazel.Runfiles
-import System.FilePath
-import Test.Tasty.HUnit
-
-
-main :: IO ()
-main = defaultMain  =<< unitTests
-
-unitTests :: IO TestTree
-unitTests = do
-    withTempFile $ \path -> do
-        darPath <- locateRunfiles (mainWorkspace </> "compiler/damlc/tests/visual-test-daml.dar")
-        dotFile <- locateRunfiles (mainWorkspace </> "compiler/damlc/tests/visual/Basic.dot")
-        return $ testGroup "making sure we do not add extra edges" [
```

Comment:
The non-golden tests still seem to be useful but are only testing the `daml-lf-reader` and not the visualize code so I would propose that we keep this, remove the golden tests and rename it to something like `//compiler/damlc/tests:daml-lf-reader`.

Reply:
Yup brought them back and renamed it is DarReaderTest